hardware/wemo/README.md

```diff
@@ -21,10 +21,14 @@ from opening the config dialog and selecting the node you want.
 
 The node accecpts the following inputs
 
- * Strings on/off
- * integers 1/0
- * boolean true/false
- * an Object like this (lights only & color control is still work in the progress) 
+ * A single value, such as
+     * Strings `on`/`off`
+     * integers `1`/`0`
+     * boolean `true`/`false`
+```
+on
```

Comment:
on ? or did you mean or ?

Reply:
It's an example of what to send to the node, so `on`. I admit it's weird that there are sub-items between the item preceding the example. Let me see if I can improve it...

Reply:
ok - so it should be titled "object"  (like integer boolean etc) - then have the example.

Reply:
actually - or without the blockquotes would work fine.

Also while you are at it... - just before the list maybe make the line
The node accecpts the following inputs   - into - The node accepts the following `msg.payload` inputs